.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file maybe shoud be gitignore, but I wanted to share a useful config for vs code. Maybe you already have configured this.

Reply:
I do have this config 👍